node-installer/tests/integration-test-k3s.sh

```diff
@@ -65,7 +76,11 @@ sudo k3s ctr images pull ghcr.io/spinkube/containerd-shim-spin/examples/spin-rus
 kubectl apply -f ./tests/workloads/workload.yaml
 
 echo "Waiting for deployment to be ready..."
-kubectl wait --for=condition=Available deployment/wasm-spin --timeout=120s
+if ! kubectl wait --for=condition=Available deployment/wasm-spin --timeout=120s; then
+  echo "Deployment failed to become ready!"
+  kubectl describe deployment wasm-spin
```

Comment:
Since we collect the logs regardless, this logging i added may be redundant

Reply:
Done